crates/topos-tce/src/app_context/api.rs

```diff
@@ -45,39 +42,6 @@ impl AppContext {
                     .await;
             }
 
-            ApiEvent::PeerListPushed { peers, sender } => {
-                let sampler = self.tce_cli.clone();
-                let gatekeeper = self.gatekeeper.clone();
-                let events = self.events.clone();
-                let api = self.api_client.clone();
-
-                spawn(async move {
-                    match gatekeeper.push_peer_list(peers).await {
-                        Ok(peers) => {
-                            info!(
-                                "Gatekeeper has detected changes on the peer list, new sample in \
-                                 creation"
-                            );
-                            if sampler.peer_changed(peers).await.is_err() {
-                                _ = sender.send(Err(RuntimeError::UnableToPushPeerList));
-                            } else {
-                                api.set_active_sample(true).await;
-                                if events.send(Events::StableSample).await.is_err() {
-                                    error!("Unable to send StableSample event");
-                                }
```

Comment:
You need to update the `active_sample` to make the service healthy and ready.
You can put it directly as true when the node starts